core/templates/dev/head/pages/exploration_editor/simple_editor_tab/questions/MultipleChoiceEditorDirective.js

```diff
@@ -0,0 +1,162 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview Directive for the a multiple choice question in the simple
+ * editor.
+ */
+
+oppia.directive('multipleChoiceEditor', [function() {
+  return {
+    restrict: 'E',
+    scope: {
+      getStateName: '&stateName',
+      getCustomizationArgs: '&customizationArgs',
+      getAnswerGroups: '&answerGroups',
+      getRawDefaultOutcome: '&defaultOutcome',
```

Comment:
Because we modify getDefaultOutcome() to make it "valid" prior to displaying it. I don't really see a cleaner way to do this...